src/components/Navigation/MobileNavigation/index.js

```diff
@@ -0,0 +1,234 @@
+import { Section } from "@commons-ui/core";
+import {
+  Grid,
+  Slide,
+  Dialog,
+  DialogActions,
+  IconButton,
+  SvgIcon,
+  DialogContent,
+} from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+import { ReactComponent as MenuCloseIcon } from "@/pesayetu/assets/menu_close.svg";
+import MenuOpenIcon from "@/pesayetu/assets/menu_open.svg";
+import DropdownSearch from "@/pesayetu/components/DropdownSearch";
+import Logo from "@/pesayetu/components/Logo";
+import Menu from "@/pesayetu/components/Menu";
+
+const useStyles = makeStyles(({ breakpoints, typography, palette }) => ({
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  section: {},
```

Comment:
We will need to customize the default padding for section on this mobile component.
Default section padding (left & right ) is: 15 
On design: Mobile has 17 left, 20 right
& tablet has 17 left, 40 right 

** You will have to re-calculate the right though (take into account the margin you have on menu icon)

Reply:
@KhadijaMahanga Ahh, okay...will do